content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>The note about using your own images is perfect for a "On your own" section at the end of the tutorial :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
👍 Added to the bottom under `## Next steps` + more learning material.